schema/CVE_Record_Format.json

```diff
@@ -303,6 +303,30 @@
                             },
                             {
                                 "required": ["version", "status", "versionType", "lessThanOrEqual"]
+                            },
+                            {
+                                "required": ["status", "versionType"],
+                                "oneOf": [
+                                    {"required": ["exactly"]},
+                                    {"required": ["inclusiveUpperBound"]},
+                                    {"required": ["exclusiveLowerBound"]},
+                                    {"required": ["inclusiveUpperBound"]},
+                                    {"required": ["exclusiveUpperBound"]}
+                                ]
+                            },
+                            {
+                                "required": ["status", "versionType", "inclusiveLowerBound"],
+                                "oneOf": [
+                                    {"required": ["inclusiveUpperBound"]},
+                                    {"required": ["exclusiveUpperBound"]}
+                                ]
+                            }
```

Comment:
Looks like a comma missing here? @darakian ?

Reply:
Got it thanks 👍